lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -123,7 +139,14 @@
     REGISTRATION_FIELDS[4:] +
     CERTIFICATE_FIELDS
 )
+"""
 
+CME_SPECIFIC_ORDER = (
```

Comment:
This is duplicated only for testing reasons, right? Please remove this if that is the case